tests/test_http.py

```
@@ -60,46 +61,50 @@ def test_configure_timeout(client: Client):
+    assert isinstance(adapter, HTTPAdapter)
+    assert adapter.max_retries.total == client.retries
+
     client.configure(retries=99)
     assert client.retries == 99
-    assert client.sess.adapters["https://"].max_retries.total == 99
+    adapter = client.sess.adapters["https://"]
+    assert isinstance(adapter, HTTPAdapter)
+    assert adapter.max_retries.total == 99
 
 
 def test_configure_status_forcelist(client: Client):
```

Ruff (0.8.0)

Missing return type annotation for public function `test_configure_status_forcelist`

Add return type annotation: `None`

(ANN201)